frontend/src/metabase/dashboard/components/DashboardActions.jsx

```diff
@@ -117,7 +118,7 @@ export const getDashboardActions = (
     );
   }
 
-  if (!isEditing && isFullscreen) {
+  if (!isEditing && isFullscreen && !hideNightModeToggle) {
```

Comment:
Hide theme toggle when loading the dashboard with a `transparent` theme.

https://github.com/metabase/metabase/pull/23482#discussion_r905142261

This is the new behavior
https://www.loom.com/share/7cf29f2c2020456db6d7b6215d96afba

Reply:
nit: can it be changed to something "truthy"/"positive"? e.g. `hasNightModeToggle` etc.